examples/dashboard/dashboard.py

```diff
@@ -330,6 +312,30 @@ def load_datasets(self,outofcore):
         else:
             raise IOError("Unknown data file type; .csv and .castra currently supported")
 
+    def create_categorical_legend(self, colormap, colornames):
+        plot_options = {}
+        plot_options['x_range'] = Range1d(start=0, end=200)
+        plot_options['y_range'] = Range1d(start=0, end=100)
+        plot_options['plot_height'] = 120
+        plot_options['plot_width'] = 190
+
+        plot_options['min_border_bottom'] = 0
+        plot_options['min_border_left'] = 0
+        plot_options['min_border_right'] = 0
+        plot_options['min_border_top'] = 0
+        plot_options['outline_line_width'] = 0
+        plot_options['toolbar_location'] = None
+
```

Comment:
Seems like `plot_options = dict( opt=val ...) would be much more concise here...